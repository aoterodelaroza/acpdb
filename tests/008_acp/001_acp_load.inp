## check: 001_acp_load.out -a1e-10
## labels: regression quick

verbose
acp load bleh ../zz_source/minis.bsip
acp load bleh2
 C l 0.2 40.0
 N l 0.2 42.0
 N s 0.3 43.0
 N p 0.4 44.0
end
acp info bleh
acp info bleh2
