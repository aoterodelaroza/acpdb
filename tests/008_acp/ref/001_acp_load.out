%% verbose
%% acp load bleh ../zz_source/minis.bsip
* ACP LOAD bleh

%% acp load bleh2
* ACP LOAD bleh2

%% acp info bleh
* INFO for ACP: bleh
+ Atoms (lmax) : H(s) B(p) C(p) N(p) O(p) F(p) Si(p) P(p) S(p) Cl(p) 
+ Number of terms : 
H: l=7 s=5 
B: l=6 s=3 p=2 
C: l=8 s=3 p=3 
N: l=6 s=2 p=3 
O: l=6 s=3 p=5 
F: l=7 s=2 p=1 
Si: l=3 s=1 p=2 
P: l=4 s=1 p=2 
S: l=5 s=2 p=3 
Cl: l=4 s=1 p=2 
1-norm = 1.50000e+01
2-norm = 2.92727e+00
inf-norm (max. abs(coef)) = 1.86200e+00

%% acp info bleh2
* INFO for ACP: bleh2
+ Atoms (lmax) : C(l) N(p) 
+ Number of terms : 
C: l=1 
N: l=1 s=1 p=1 
1-norm = 1.69000e+02
2-norm = 8.45518e+01
inf-norm (max. abs(coef)) = 4.40000e+01

